src/SelectPanel/SelectPanel.tsx

```diff
@@ -141,9 +146,11 @@ export function SelectPanel({
     return {
       sx: {m: 2},
       contrast: true,
+      leadingVisual: SearchIcon,
+      'aria-label': inputLabel,
```

Comment:
:accessibility: A11y question: Should we use a visually hidden label instead of aria-label for the filter input?

Reply:
I'd say it doesn't matter as much as they both will announce the same. I'd say one benefit that you could potentially gain with a visually hidden label and `aria-labelledby` is that it _might_ translate better with automated translation tools versus an `aria-label`, but I believe support has gotten better since I last checked. Other than that, I think using `aria-label` is fine.